CRM/Event/Form/FormBlock.php

```diff
@@ -0,0 +1 @@
+<?php
```

Comment:
@eileenmcnaughton why is this needed?

Reply:
arg - the extra files would be part of the bigger fix - but the 2 small changes you might like to pull into your PR & I'll close this